### DISCUSSION TITLE
Custom Order via applyOrderings not longer working

### DISCUSSION THREAD

**Discussion by StanBarrows** [laravel/nova-issues#3986] [Q&A] [answered]
In Nova 3 I've used the attached example to order my resources. This is not longer possible within Nova 4. Any ideas to achieve this?

**Resource**
`    public static $orderBy = ['identification' => 'asc'];`

**Custom Trait**
```
   protected static function applyOrderings($query, array $orderings)
    {
        $orderings = static::$orderBy;
        return parent::applyOrderings($query, $orderings);
    }
}
```

**Comment by StanBarrows** [comment, marked as answer]
It's working, maybe a cache issue.

```
    protected static function applyOrderings($query, array $orderings)
    {
        $orderings = ['name' => 'asc'];
        return parent::applyOrderings($query, $orderings);
    }
```

**Reply by ekazda** [reply]
Are you able to share all the code you wrote to implement this? I'm trying to do this as well but I cannot seem to get my custom trait to actually fire.
